boards/qemu/riscv32e/board.yml

```diff
@@ -1,5 +1,6 @@
 board:
   name: qemu_riscv32e
+  full_name: QEMU Emulation for RISCV32E Emulation
```

Comment:
why Emulation twice in this name ?

To be consistent with other Qemu boards:
```suggestion
  full_name: QEMU Emulation for RISCV32E
```

Reply:
definitely a typo! :) good catch - thank you

Reply:
@kartben still present on main, will you do a followup PR ?
https://github.com/zephyrproject-rtos/zephyr/blob/6300e1bc253531b992c868424543cde112189915/boards/qemu/riscv32e/board.yml#L3